jmsSource/src/test/java/io/vantiq/extsrc/jmsSource/NullMessageHandler.java

```diff
@@ -0,0 +1,22 @@
+package io.vantiq.extsrc.jmsSource;
+
+import java.util.Map;
+
+import javax.jms.Message;
+import javax.jms.Session;
+
+import io.vantiq.extsrc.jmsSource.communication.messageHandler.MessageHandlerInterface;
+
+public class NullMessageHandler implements MessageHandlerInterface {
```

Comment:
Seems like we will also need an example of a message handler that doesn't return the required properties and/or doesn't create a valid JMS message